src/frontend/apps/desk/src/features/menu/MenuItems.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button } from '@openfun/cunningham-react';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+import React, { useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import { Box } from '@/components/';
+import { useCunninghamTheme } from '@/cunningham';
+import { SVGComponent } from '@/types/components';
+
+import { Tooltip } from './Tooltip';
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #ffffff33;
+  &[aria-current='page'] {
+    color: #ffffff;
+  }
+`;
+
+interface MenuItemProps {
+  Icon: SVGComponent;
+  label: string;
+  href: string;
+  width?: number;
+}
+
+const MenuItem = ({ Icon, label, href, width = 38 }: MenuItemProps) => {
+  const { t } = useTranslation();
+  const pathname = usePathname();
+  const { colorsTokens } = useCunninghamTheme();
+  const buttonRef = useRef(null);
+  const [isTooltipOpen, setIsTooltipOpen] = useState(false);
+
+  const isActive = pathname === href;
+  const color = isActive ? colorsTokens()['primary-600'] : '#ffffff55';
+  const background = isActive ? colorsTokens()['primary-300'] : undefined;
+
+  return (
+    <StyledLink
+      href={href}
+      aria-current={isActive ? 'page' : undefined}
+      ref={buttonRef}
+      onMouseOver={() => setIsTooltipOpen(true)}
+      onMouseOut={() => setIsTooltipOpen(false)}
+    >
+      <Box
+        className="m-st p-t"
+        as="li"
+        $justify="center"
+        $css={`
+          & > button { padding: 0};
+          transition: all 0.2s ease-in-out
+        `}
+        $background={background}
+        $radius="10px"
+      >
+        <Button
+          aria-label={t(`{{label}} button`, { label })}
+          icon={
+            <Box $color={color}>
+              <Icon
+                width={width}
+                aria-label={t(`{{label}} icon`, { label })}
+                style={{
+                  transition: 'color 0.2s ease-in-out',
+                }}
+              />
+            </Box>
+          }
+          color="tertiary"
+          className="c__button-no-bg p-0 m-0"
+          style={{ flexDirection: 'column', gap: '0' }}
+        ></Button>
+      </Box>
+      {isTooltipOpen && (
+        <Tooltip
+          buttonRef={buttonRef}
+          label={label}
+          backgroundColor={isActive ? '#ffffff' : '#161616'}
+          textColor={isActive ? '#000000' : '#ffffff'}
```

Comment:
the `isActive ?` ternary check is duplicated at least 4 times. I would suggest introducing a `useMemo` with dependencies on `pathname` and `href` which returns all colors of the page.

Reply:
brainsto: should we facilitate color management, instead of managing hardcoded and possibly duplicated literals which contains hexa code?

Reply:
What do you suggest ?  These colors are not really part of the theme I have the feeling.

Reply:
```
const activePalette = {
    icon: {
        color: colorsTokens()['primary-600'],
        background: colorsTokens()['primary-300']
    }
    tooltip: {
        color: 'black',
        background: 'white'
    }
};

const defaultPalette = {
    icon: {
        color: '#ffffff55',
    }
    tooltip: {
        color: 'white',
        background: '#161616'
    }
};

const palette = isActive ? activePalette : defaultPalette


console.log(palette.icon.color)
```

Reply:
Might be overengineered ! Btw, the tooltip color is all the time black in your fixup.

Reply:
https://github.com/numerique-gouv/people/commit/befd84f2ad95b00881fcd6c7c486e74110c5275c